src/Components/Modal/Modal.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import './Modal.scss';
+
+class Modal extends React.Component {
+  render() {
+    const { open, close, header } = this.props;
+
+    return (
+      <>
+        <div className={open ? 'modal openModal' : 'modal'}>
+          {open ? (
```

Comment:
```suggestion
          {open && (
```
true인 경우만 표현한다면 삼항연산자 대신 `&&`를 사용해줄 수 있습니다!

Reply:
&&를 사용하면 false 로직을 적어 줄 필요가 없군요! 감사합니다